doc/command-usage.rst

```diff
@@ -26,6 +26,13 @@ Build a Charliecloud image from Dockerfile and unpack it into a directory.
 
 .. include:: ./ch-build2dir_desc.rst
 
+ch-builder2squash
++++++++++++++++++
+
+Flatten a builder image into a Charliecloud SquashFS file.
+
+.. include:: ./ch-builder2squash_desc.rst
+
 ch-builder2tar
 ++++++++++++++
```

Comment:
Shouldn't this be shoveled now?

Reply:
Lots of the existing scripts will get the shovel. I can add deprecation notices to them.